packages/synapse-react-client/src/components/GoogleAnalytics/GoogleAnalytics.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { useGoogleAnalytics } from '../../utils/hooks'
+
+export type GoogleAnalyticsProps = {
+  measurementId?: string
+}
+/**
+ * GoogleAnalytics is a component wrapper for the useGoogleAnalytics() hook.
+ * This is exported for use in the Synapse.org GWT codebase.
+ * @param props
+ */
+export const GoogleAnalytics: React.FunctionComponent<GoogleAnalyticsProps> = (
```

Comment:
I really like the idea to capture all GA logic in one hook, then reuse the hook (via this component) in SWC